package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@readme/emojis": "^4.0.0",
-    "@readme/syntax-highlighter": "^10.13.0",
+    "@readme/syntax-highlighter": "^10.13.2",
```

Comment:
You might have that codemirror problem again with this one when you pull it over into the site, I'm not sure. Hopefully not.

Reply:
Oh yea, totally forgot about that.